src/charts/series-chart.ts

```diff
@@ -1,4 +1,4 @@
-import {ascending} from 'd3-array';
+import {ascending, Primitive} from 'd3-array';
```

Comment:
Primitive doesn't look like an export of d3-array?

Reply:
It is exported from @types/d3-array.